### ISSUE TITLE
<dl> structure has invalid HTML in the database incident cards

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/pressfreedomtracker.us#1326]
![image](https://user-images.githubusercontent.com/9530293/160896696-15818117-43bf-4144-a86c-6a488b5e708d.png)

The above is what we have now. A <dt> element cannot have heading content. We can probably just wrap the entire thing inside a div.
